crowd_anki/representation/note.py

```diff
@@ -38,8 +41,8 @@ def from_collection(cls, collection, note_id, note_models):
         return note
 
     @classmethod
-    def from_json(cls, json_dict):
-        note = Note()
+    def from_json(cls, json_dict, config: ConfigSettings = None):
+        note = Note(config=config)
```

Comment:
why do we need these?

Reply:
Well I added an optional config parameter to the Note constructor, and these are the only functions which ever call the Note constructor. They are used throughout the code, as the default way to make notes, so I thought they also needed the ability to have this config item passed to them :thinking: No one is ever going to simple call Note()

Reply:
eh, this argument have some merit, but I'm more of a mind that the code that is not used should not be there. Do you envision this being used any time soon?

Reply:
Nope, not in the slightest! I shall remove it then :wink: